plugins/woocommerce-blocks/assets/js/base/stores/add-to-cart.ts

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
Nitpick, but I wonder if we should namespace the stores in the folder structure like `stores/cart/add-to-cart.ts`. It might be easier to organize and work with as we add more stores.